doc/rados/configuration/mon-lookup-dns.rst

```diff
@@ -2,15 +2,19 @@
 Looking up Monitors through DNS
 ===============================
 
-Since version 11.0.0 RADOS supports looking up Monitors through DNS.
+Since Ceph version 11.0.0 (Kraken), RADOS has supported looking up monitors
+through DNS.
 
-This way daemons and clients do not require a *mon host* configuration directive in their ceph.conf configuration file.
+The addition of the ability to look up monitors through DNS means that daemons
+and clients do not require a *mon host* configuration directive in their
+``ceph.conf`` configuration file.
 
-Using DNS SRV TCP records clients are able to look up the monitors.
+By making use of a DNS update, clients and daemons can be made aware of changes
```

Comment:
Suggest 

With a DNS update

Reply:
I'm chary of with-clauses, but if your fresh eyes tell you that it reads better, I'll take the benefit of those fresh eyes.